docs/cookbook/agents/sql_agent.md

```diff
@@ -0,0 +1,336 @@
+# Generate SQL with LLM
+
+In this recipe, we will be using OpenAI GPT-4o-mini to act as a co-pilot for a Database Admin. While LLMs are powerful and do pretty well at transforming laymen queries into SQL queries, it is still dangerous to do so without supervision. This recipe will have no guardrails for mutable operations and is purely for getting started.
+
+??? tip "Mirascope Concepts Used"
+
+    - [Prompts](../../learn/prompts.md)
+    - [Calls](../../learn/calls.md)
+    - [Tools](../../learn/tools.md)
+    - [Dynamic Configuration](../../learn/dynamic_configuration.md)
+    - [Async](../../learn/async.md)
+    - [Evals](../../learn/evals.md)
+    - [Agents](../../learn/agents.md)
+
+## Setup
+
+We will be using SQLite but this example will work for any common SQL dialect, such as PostgreSQL, MySQL, MSSQL, and more.
+
+```python
+pip install sqlite3
+```
+
+## Setup SQL Database
+
+Replace this part with whichever SQL dialect you are using, or skip if you have a database setup already.
```

Comment:
nit: "setup" should be "set up" in this case I believe, could instead say "if you have already setup a database"